SU2_CFD/src/solvers/CPassiveScalarSolver.cpp

```diff
@@ -236,10 +238,41 @@ CPassiveScalarSolver::~CPassiveScalarSolver(void) {
 void CPassiveScalarSolver::Preprocessing(CGeometry *geometry, CSolver **solver_container, CConfig *config,
          unsigned short iMesh, unsigned short iRKStep, unsigned short RunTime_EqSystem, bool Output) {
 
+  unsigned long n_not_in_domain     = 0;
+
   const bool implicit = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
-  const bool muscl = config->GetMUSCL_Scalar();
-  const bool limiter = (config->GetKind_SlopeLimit_Scalar() != NO_LIMITER) &&
-                       (config->GetInnerIter() <= config->GetLimiterIter());
+  const bool muscl    =  config->GetMUSCL_Scalar();
+  const bool limiter  = (config->GetKind_SlopeLimit_Scalar() != NO_LIMITER) &&
+                        (config->GetInnerIter() <= config->GetLimiterIter());
+
+  for (auto i_point = 0u; i_point < nPoint; i_point++) {
+
+    CFluidModel * fluid_model_local = solver_container[FLOW_SOL]->GetFluidModel();
+    su2double * scalars = nodes->GetSolution(i_point);
+
+    su2double Temperature = solver_container[FLOW_SOL]->GetNodes()->GetTemperature(i_point);
+
+    n_not_in_domain += fluid_model_local->SetTDState_T(Temperature, scalars); /*--- first argument (temperature) is not used ---*/
```

Comment:
ah, I guess for lookuptables this returns how many points were outside the triangulation @danielmayer ?